debezium-core/src/main/java/io/debezium/pipeline/source/snapshot/incremental/AbstractIncrementalSnapshotContext.java

```diff
@@ -73,6 +73,10 @@
      */
     private Object[] maximumKey;
 
+    private IncrementalSnapshotSchema schema;
+
+    private boolean schemaVerified;
```

Comment:
Can we rename the flag variable. From the name it is not clear whether the verification was executed or whether the verification has passed.